src/app/modules/dashboard/components/dashboard/dashboard.component.ts

```diff
@@ -23,22 +25,25 @@ export class DashboardComponent implements OnInit {
     }
 
     ngOnInit() {
-        if (this.profileService.profile) {
-            this.profileService.getSavedSearches(this.profileService.profile.userId).subscribe(x => {
+        if (localStorage.getItem('profile')) {
```

Comment:
This must be on profileService
We just call profileService.isAuthorized() to check if user logged

Reply:
I will add isAuthorized function